client_secrets.json

```diff
@@ -0,0 +1 @@
+{"web": {"redirect_uris": ["https://paste/oidc_callback"], "token_uri": "https://idp/openidc/Token", "auth_uri": "https://idp/openidc/Authorization", "client_id": "myclient", "client_secret": "thesecret", "userinfo_uri": "https://idp/openidc/UserInfo", "token_introspection_uri": "https://idp/openidc/TokenInfo", "issuer": "https://idp/openidc/"}}
```

Comment:
Doesn't seem like this secrets file should be versioned. Can you delete this file and instead document the shape of the secrets object in the README?